packages/selleo-design-core/src/Avatars.tsx

```diff
@@ -0,0 +1,74 @@
+import { h } from "preact";
+
+import cx from "classnames";
+
+export type AvatarProps = {
+  hideName?: boolean;
+  imageSize?: number;
+  image?: h.JSX.Element | null;
+  username?: string;
+};
+
+export function SmallAvatar({
+  hideName = false,
+  imageSize = 24,
+  image,
+  username,
+}: AvatarProps) {
+  const imageClasses = cx(
+    `w-[${imageSize}px] h-[${imageSize}px] rounded-full overflow-hidden min-w-fit flex justify-center items-center`,
```

Comment:
tailwindclasses should not be interpolated since the compiler won't know which values to extract into stylesheet

Reply:
Right, done